locales/tl-PH.json

```
@@ -0,0 +1,630 @@
+    "status_not_found": "Hindi mahanap ang post",
+    "unsupported_file_format": "Hindi suportado ang file format"
+  },
+  "help": {
+    "build_preview": {
+      "desc1": "Ikaw ay kasalukuyang nakakakita ng preview version ng Elk mula sa community - {0}.",
+      "desc2": "Maaaring mayroon itong hindi pa na-review o kaya malicious changes.",
+      "desc3": "Huwag mag-login gamit ang iyong tunay na account.",
+      "title": "Preview deploy"
+    },
+    "desc_highlight": "Maaaring mayroon kang makikitang mga bug at kulang na features dito at doon.",
+    "desc_para1": "Salamat sa iyong interes sa pagsubok ng Elk, aming work-in-progress na Mastodon web client!",
+    "desc_para2": "Dedikado kami sa pag-develop at pagpapabuti dito sa mga susunod na panahon.",
+    "desc_para3": "Para mapalago ang pag-develop, maaari kang mag-sponsor sa Team sa pamamagitan ng GitHub Sponsors. Sana magustuhan mo ang Elk!",
+    "desc_para4": "Ang Elk ay Open Source. Kung nais mong tumulong sa pagsubok, magbigay ng feedback, o mag-contribute,",
+    "desc_para5": "makipag-ugnayan sa amin sa GitHub",
+    "desc_para6": "at makilahok.",
+    "footer_team": "Ang Elk Team",
+    "title": "Ang Elk ay nasa Preview!"
+  },
+  "language": {
+    "search": "Hanapin"
+  },
+  "list": {
+    "add_account": "Magdagdag ng account sa listahan",
+    "cancel_edit": "kanselahin ang pag-edit",
+    "clear_error": "Burahin ang error",
+    "create": "lumikha",
+    "delete": "Tanggalin ang listahang ito",
+    "delete_error": "Mayroong error sa pagtanggal ng listahan",
+    "edit": "I-edit ang listahang ito",
+    "edit_error": "Mayroong error sa pag-update ng listahan",
+    "error": "Mayroong error sa paglikha ng listahan",
+    "error_prefix": "Error: ",
+    "list_title_placeholder": "Pamagat ng listahan",
+    "modify_account": "Baguhin ang mga listahan sa account",
+    "remove_account": "Tanggalin ang account sa listahan",
+    "save": "I-save ang mga pagbabago"
+  },
+  "magic_keys": {
+    "dialog_header": "Mga shortcut sa keyboard",
+    "groups": {
+      "actions": {
+        "boost": "Mag-boost",
+        "command_mode": "Mode ng Command",
+        "compose": "Mag-compose",
+        "favourite": "Paborito",
+        "title": "Mga aksyon"
+      },
+      "media": {
+        "title": "Media"
+      },
+      "navigation": {
+        "go_to_home": "Tungo sa Home",
+        "go_to_notifications": "Tungo sa mga Notification",
+        "next_status": "Susunod na status",
+        "previous_status": "Nakaraang status",
+        "shortcut_help": "Tulong sa shortcut",
+        "title": "Pag-navigate"
+      }
+    },
+    "sequence_then": "pagkatapos"
+  },
+  "menu": {
+    "add_personal_note": "Magdagdag ng personal na note sa {0}",
+    "block_account": "I-block ang {0}",
+    "block_domain": "I-block ang domain ng {0}",
+    "copy_link_to_post": "Kopyahin ang link ng post na ito",
+    "copy_original_link_to_post": "Kopyahin ang orihinal na link ng post na ito",
```

```suggestion
    "copy_link_to_post": "Kopyahin ang link,
    "copy_original_link_to_post": "Kopyahin ang orihinal na link,
```

I think we should shorten the items in this menu

---

oops no close quotation. ako na mag fix kabayan